devel/got/Portfile

```
@@ -20,15 +20,16 @@ long_description    Game of Trees is a version control system which \
                     repository.
 homepage            https://gameoftrees.org/
 master_sites        ${homepage}releases/portable/
-checksums           rmd160 c1e1a71c65dccd932c0ce2a2ea1081faa92a8016 \
-                    sha256 79b15eb508601018f2ddaab74df2bdbde79ebdb992004bfd91a52886c9ecae55 \
-                    size 1146628
+checksums           rmd160 d4bf70c788b6c8b66735f030bddd58e6a4eafc50 \
+                    sha256 1da342c606e5d1ec2f23dea3afdfc1809a61aa8402e4fd9ab63e5ae756e3f7d7 \
+                    size 1265096
 depends_lib         path:lib/libssl.dylib:openssl \
                     port:ncurses \
                     port:ossp-uuid \
                     port:libevent
 depends_build       port:bison \
                     port:pkgconfig
+configure.args --enable-gotd --with-gitwrapper-git-libexec-path --with-gotd-empty-path
```

Can this be changed to the usual indentation?
```suggestion
configure.args      --enable-gotd \
                    --with-gitwrapper-git-libexec-path \
                    --with-gotd-empty-path
```